setup.cfg

```diff
@@ -1,5 +1,10 @@
 [metadata]
-description-file = README.md
+long_description = file: README.md
+long_description_content_type = text/markdown
+
+[options.entry_points]
+console_scripts =
+    wikidataintegrator-publication = wikidataintegrator.wdi_helpers.publication:main
```

Comment:
This makes it possible to run this python file directly as a CLI command from the shell like:

```sh
$ wikidataintegrator-publication 2006.08589 --source arxiv --idtype arxiv instead
```

If you don't like the name, the thing before the `=` can be set to whatever you want.